typings/single-spa.d.ts

```diff
@@ -122,7 +122,7 @@ declare module "single-spa" {
 
   type Application<ExtraProps = {}> =
     | LifeCycles<ExtraProps>
-    | ((config: ExtraProps & AppProps) => Promise<LifeCycles<ExtraProps>>);
+    | ((config: ExtraProps & AppProps) => Promise<LifeCycles<ExtraProps>> | Promise<System.Module>);
```

Comment:
Promise<System.Module> should to be first option

![Captura de Tela 2022-09-22 às 23 14 28](https://user-images.githubusercontent.com/34426407/191880881-16498ed8-8783-4756-b462-3221ea41f018.png)

![Captura de Tela 2022-09-22 às 23 12 59](https://user-images.githubusercontent.com/34426407/191880893-e27cd6ae-553c-42f4-adba-d5fba748b0bf.png)

Reply:
the order dont have problem!
in you case the problem apears because you comit mistakes, you didnts use this
```JS
((config: ExtraProps & AppProps) => Promise<LifeCycles<ExtraProps>> |  Promise<System.Module>) // <- Look  Promise<System.Module> is into ()
```
in your case you put out like this:
```JS
 ((config: ExtraProps & AppProps) => Promise<LifeCycles<ExtraProps>>)  |  Promise<System.Module> // now we have a problem
```

=)

Reply:
Great!!!